### PR TITLE
Version 4.2.0

### PR DESCRIPTION
**Pull request by nulltoken**
## PR Checklist

- [x] I have run `npm test` locally and all tests are passing.
- [ ] I have added/updated tests for any new behavior.
- [ ] If this is a significant change, an issue has already been created where the problem / solution was discussed: [N/A, or add link to issue here]

## PR Description

Release new version with #150 and #152 contributed by @roskh

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "target": "es2019", /* Minimum supported Node version is ^12.20 => cf. https://stackoverflow.com/a/59787575/335418 */
     "module": "commonjs", /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
     "lib": [
-      "es2020"
+      "es2020",
+      "DOM"
```
Comment (nulltoken) [inline tsconfig.json:13]: @poveden Required by the update to superagent which references the type 'Blob'

File: src/lib/oauth2-service.ts [hunk 2]
```diff
@@ -20,11 +20,13 @@
  */
 
 import { IncomingMessage } from 'http';
+import { URL } from 'url';
```
Comment (nulltoken) [inline src/lib/oauth2-service.ts:23]: @poveden With the latest update, TypeScript started to complain that it didn't know about that type. Had to reference it explicitly

**Review by poveden — APPROVED** [review]

**Comment by nulltoken** [comment]
@poveden Merged! Can you plan the release please?

**Comment by poveden** [comment]
@nulltoken It is done. Sorry for the delay.

**Pull request merged**